cassandane/Cassandane/Net/SMTPServer.pm

```diff
@@ -74,7 +74,7 @@ sub rcpt_to {
     $Self->{_rcpt_to_count}++;
     if ($Self->{_rcpt_to_count} > 10) {
         $Self->send_client_resp(550, "5.5.3 Too many recipients");
-    } elsif ($To =~ /\@fail\.to\.deliver$/i) {
+    } elsif ($To =~ /[<>]/ || $To =~ /\@fail\.to\.deliver$/i) {
```

Comment:
Ah yep -- make the mock reject the bad rcpt like a real server would, and then if it's no longer getting rejected, we're no longer sending bad rcpt's, nice